com.unity.ml-agents/Runtime/Inference/TensorGenerator.cs

```diff
@@ -94,36 +94,46 @@ public interface IGenerator
         public void InitializeObservations(List<ISensor> sensors, ITensorAllocator allocator)
         {
             // Loop through the sensors on a representative agent.
-            // For vector observations, add the index to the (single) VectorObservationGenerator
-            // For visual observations, make a VisualObservationInputGenerator
+            // For vector observations, add the index to the (single) ObservationGenerator
+            // For visual observations, make a NonVectorObservationInputGenerator
```

Comment:
Definitely need to remove the reference to `NonVectorObservationInputGenerator` since that no longer exists. How about this?
```suggestion
            // All vector observations use a shared ObservationGenerator since they are concatenated.
            // All other observations use a unique ObservationInputGenerator
```